scipy/integrate/_ivp/ivp.py

```diff
@@ -468,6 +472,8 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
      1.11088891e-01 1.11098890e+00 1.11099890e+01 4.00000000e+01]
     >>> print(sol.sol(sol.t_events[1][0]))
     [100.   0.]
+    >>> print(sol.y_events)
+    [array([[-5.68434189e-14, -1.00000000e+01]]), array([[1.00000000e+02, 1.77635684e-15]])]
```

Comment:
Don't use print, just do as if you're in an interpreter: `>>> sol.y_events`. 
This should take care of refguide_check complaints.

Reply:
Thanks @ev-br.  Are you suggesting to remove print from the docstring entirely?  My usage is consistent with the entire `solve_ivp` docstring, but I'm willing to clean it up here if needed.

Or, are you suggesting just to do `>>>  sol.y_events` to get the correctly formatted return value, but keep print statements in the docstring?

Reply:
Now that I've written this down and reread what I wrote, the second option doesn't seem right.  So, I'm guessing the path forward is to remove all prints from the `solve_ivp` docstring?

Reply:
I would write `>>> sol.t_events` in the docstring, run `$ python runtests --refguide-check -s optimize` locally, and copy-paste the output from the failure reports. This normally allows to forget about the whole doctesting from that point on.

Reply:
Thanks for the pointers!  I can do this for this PR, but I'm feeling that this whole docstring would be inconsistent.  Is it okay if I remove all prints in this PR even though it is unrelated to this change? Or should this be a separate PR on documentation cleanup?

Reply:
Maybe do the bare minimum to get the CI green, and then a separate DOC PR.

Reply:
The CI **is** green. I'm going to resolve this conversation and leave as is, assuming that the current format is correct, since the CI checks out.

I have a lot of struggle with setting up a full fledged scipy dev environment, so I can't run these tools locally.